Source/Charts/Charts/PieChartView.swift

```diff
@@ -639,4 +642,28 @@ open class PieChartView: PieRadarChartViewBase
             }
         }
     }
+    
+    /// The offset value that is used for shifting the data label outside of the center of chart.
+    /// **default**: 0.0
+    open var dataLabelOffset: CGFloat
+        {
+        get
+        {
+            return _dataLabelOffset
+        }
+        set
+        {
+            _dataLabelOffset = newValue
+            
+            if _dataLabelOffset > 1.0
```

Comment:
still, we need some justifications for max to 1.0. Why not suport > 1.0?

Reply:
Max is kept at 1.0 To limit the distance of labels if we allow it to me more than it will go vary far form the visible chart view

Reply:
@jjatie what you think the multiplier limit?

Reply:
My Idea behind this was it will be in the ration of the pie chart radius so it will be easy to move them according the chart size for the user.

Reply:
@hb2708 I see. However, as a generic feature that goes into the library, we need to make it flexible?
we are having 
```swift
 chart.radius + chart.radius * chart.dataLabelOffset
```
here so it implies the radius will be in range [radius, 2*radius], which would work in your scenario but for other users it becomes kind of restriction.

Changing to a simple absolute offset remove that restriction, and if you want to limite the range, you can just pass 2*radius for the offset.

Reply:
Ok,

Let me try to explain my approach again.
If we do not pass the dataLabelOffset then it will be 0 by default.
So data labels will be there at default position as per original library implementation.

So by using `chart.radius + chart.radius * chart.dataLabelOffset` will move the position of the data labels only SO it will move form the default position to exactly same Distance outside the chart. 

And it will limit the max distance so it will not go too away form the chart so it will be a good UX